shared/chat/inbox/row/small-team/bottom-line.js

```diff
@@ -49,6 +60,20 @@ class BottomLine extends PureComponent<Props> {
           </Text>
         </Box>
       )
+    } else if (youAreReset) {
+      content = (
+        <Text
+          type="BodySmallSemibold"
+          backgroundMode="Terminal"
+          style={{
+            color: globalColors.red,
+            fontSize: 11,
+            lineHeight: lineHeight(14),
+          }}
+        >
+          You have to be let back in.
```

Comment:
:sheep: this is pretty hard to read if the conversation is selected
<img width="240" alt="image" src="https://user-images.githubusercontent.com/11968340/34167062-a9d083f8-e4ae-11e7-8438-c27535f5cc31.png">